### ISSUE TITLE
"Clear" works incorrectly

### ISSUE THREAD

**Issue by vaskinyy** [sbaechler/django-multilingual-search#8]
**Task:** fully clear my index.
**Expected:** Haystack deletes an index, when I don't specify models to delete.
**Current:** 
Haystack `clear` method in the base class of `ElasticsearchMultilingualSearchBackend` has the following code:
```
            if models is None:
                self.conn.indices.delete(index=self.index_name, ignore=404)
                # ...
            else:
                models_to_delete = []
                # ...

```
The `else:` path of the code is executed when I specify no parameters in the child method wich has `models=[]` as a default.
Eventualy it hits another bug in haystack that method `delete_by_query` is outdated, but this is another story.
**Solution:** Set as a default parameter`models=None`

**Comment by vaskinyy** [comment]
I'm going to PR this

**Comment by vaskinyy** [comment]
Ooops, turns out it is already fixed

**Comment by sbaechler** [comment]
I have released version 2.4.1 which includes the fix.
